content_scripts/hints.js

```diff
@@ -585,7 +585,10 @@ function createHints() {
                 active = false;
             }
 
-            if (shiftKey && window.navigator.userAgent.indexOf("Firefox") !== -1) {
+            if (shiftKey && runtime.conf.hintShiftNonActive) {
+                tabbed = true;
+                active = false;
+            } else if (shiftKey && window.navigator.userAgent.indexOf("Firefox") !== -1) {
                 // mouseButton does not work for firefox in mouse event.
                 tabbed = true;
                 active = true;
```

Comment:
tabbed = shiftKey;
            active = runtime.conf.hintShiftNonActive;

Maybe this is enough, the special handling for Firefox here is only to let Firefox to call `openLink` RUNTIME instead of `dispatchMouseEvent`, since mouseButton does not work for firefox in mouse event.

Reply:
I tried what you suggested (in place of the `if (shiftKey && runtime.conf.hintShiftNonActive) { } else` block) while it works for Firefox, it doesn't work for Chrome.

On Chrome, `tabbed` is never set to `true`, so it reaches the [`dispatchMouseEvent()` branch](https://github.com/brookhong/Surfingkeys/pull/1159/files/134c98d7caa06de74ed5d852e7c68fcb4f0efe2a#diff-b535bb33f047340e9cb9ca177ed0954fR605) with no knowledge of what `active` or `runtime.conf.hintShiftNonActive` are. Effectively, `hintShiftNonActive` won't work on Chrome with that implementation (tested).

I figured this is probably not what you want, to maintain feature parity between the two browsers.